opendata-ogc-features/values.yaml

```diff
@@ -44,3 +44,4 @@ environment:
         key: password
         name: georchestra-database-geodata-secret
 tolerations: []
+ogcFeatConfig: []
```

Comment:
should be an object, not an array ? `{}` instead ?

Reply:
good catch